kv/etcd/store.go

```diff
@@ -71,17 +71,15 @@ func NewStore(etcdKV clientv3.KV, etcdWatcher clientv3.Watcher, opts Options) (k
 			diskReadError:  scope.Counter("disk-read-error"),
 		},
 	}
+	etcdWatchOpts := []clientv3.OpOption{clientv3.WithCreatedNotify()}
```

Comment:
Do we need created notify? Should we make this an option as well and disable it by default as well?